### PR TITLE
Add Beryl feeds

### PR DESCRIPTION
**Pull request by seanwiseman**
Add https://beryl.cc/ feeds. 🚲 

(I am a dev there)

### CODE REVIEW

File: systems.csv [hunk 1]
```diff
@@ -61,6 +61,9 @@ ES,Bilbaobizi (Bilbao),"Bilbao, ES",nextbike_bo,https://www.bilbaobizi.bilbao.eu
 ES,Sitycleta (Las Palmas),"Las Palmas de Gran Canaria, ES",nextbike_el,https://www.sitycleta.com/xx/laspalmas/,https://gbfs.nextbike.net/maps/gbfs/v1/nextbike_el/gbfs.json
 FR,Libélo,"Valence, FR",libelo,http://www.citea.info/presentation/?rub_code=92,https://valence.publicbikesystem.net/ube/gbfs/v1/gbfs.json
 GB,BelfastBikes,"Belfast, GB",nextbike_bu,https://www.belfastbikes.co.uk/xx/belfast/,https://gbfs.nextbike.net/maps/gbfs/v1/nextbike_bu/gbfs.json
+GB,Beryl - BCP,"Bournemouth, Christchurch, Poole, GB",beryl_bc,https://www.beryl.cc,http://gbfs.basis-pdn.bike/BCP/gbfs.json
+GB,Beryl - Hereford,"Hereford, GB",beryl_he,https://www.beryl.cc,http://gbfs.basis-pdn.bike/Hereford/gbfs.json
+GB,Beryl - London,"London, GB",beryl_lo,https://www.beryl.cc,http://gbfs.basis-pdn.bike/London/gbfs.json
```
Comment (bddq) [inline systems.csv:66]: It's too bad that your GBFS files are not hosted under an HTTPS url.

Comment (seanwiseman) [reply systems.csv:66]: This is down to an additional load balancer being required in order to allow `ssl` redirects for the hosting of these data files. Should we need to in the future we can make the appropriate infrastructure changes.

If need be users can also get it via googles domain over `ssl` e.g.:

https://storage.googleapis.com/gbfs.basis-test.bike/BCP/free_bike_status.json

**Review by jcn — CHANGES_REQUESTED** [review]
@seanwiseman The GBFS spec notes that `system_id` "should be globally unique (even between different systems) and it is currently up to the publisher of the feed to guarantee uniqueness."

Currently all of your systems report a system_id as `beryl_en` where they should instead be something like `beryl_bcp` and `beryl_london` to ensure that consumers can distinguish between the feeds internally.

**Comment by seanwiseman** [comment]
> @seanwiseman The GBFS spec notes that `system_id` "should be globally unique (even between different systems) and it is currently up to the publisher of the feed to guarantee uniqueness."
> 
> Currently all of your systems report a system_id as `beryl_en` where they should instead be something like `beryl_bcp` and `beryl_london` to ensure that consumers can distinguish between the feeds internally.

Hi @jcn , this is now implemented.

**Review by jcn — APPROVED** [review]

**Pull request merged**